init.d/imunify-nsq.init.d

```diff
@@ -43,7 +43,7 @@ start() {
         cd /
         close_fds
         export GOGC=10
-        setsid $exec --data-path /var/lib/imunify-nsqd --use-unix-sockets --tcp-address /var/run/imunify-nsqd.sock --http-address /var/run/imunify-nsqd-http.sock &
+        setsid $exec --data-path /var/lib/imunify-nsqd --max-bytes-per-queue 104857600 --use-unix-sockets --tcp-address /var/run/imunify-nsqd.sock --http-address /var/run/imunify-nsqd-http.sock &
```

Comment:
I suggest to use 1/10 for bytes per file to rollout to be sure that we removing only last 10% when it needed.

Reply:
yeap, I'll set -max-bytes-per-file 100mb + 10mb per file?

Reply:
ok